soccer/gameplay/tactics/coordinated_pass.py

```diff
@@ -66,7 +66,8 @@ def __init__(self,
         if skillkicker == None:
             skillkicker = (skills.pivot_kick.PivotKick(), lambda x: x ==
                            skills.pivot_kick.PivotKick.State.aimed or x ==
-                           skills.pivot_kick.PivotKick.State.kicking)
+                           skills.pivot_kick.PivotKick.State.kicking or x ==
+                           behavior.Behavior.State.completed)
```

Comment:
Looks good to me, this is checking both kicking and completed because we found a case where kicking was skipped I believe.